AutoscoperM/AutoscoperMLib/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
```suggestion
sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)))
```

Is this required to support using the extension by simply adding the module path through the settings ?

Reply:
This allows me to import the library code by doing 
```python
from AutoscoperMLib import *
```
instead of 
```python
from .AutoscoperMLib import *
```

Although I am not sure if this is only needed when built from source or if it is also needed for distribution.

Reply:
The corresponding sys.path update have been removed as those should not be needed.

Since `AutoscoperMLib` is its own package, import should be done using `from AutoscoperMLib import Something`, relative import are not expected to work.